src/pages/Lend/LendPage.vue

```diff
@@ -6,16 +6,19 @@
 				<p class="small-12 xlarge-9">Each Kiva loan helps people build a better
 				future for themselves and their families.</p>
 			</div>
+			<GridLoanCard />
```

Comment:
this will work for now, but probably we need a wrapping div here that is something like loan-cards-container with a v-for to iterate over the returned loans.